src/app/modules/project/overview/project/project.component.scss

```diff
@@ -34,9 +34,13 @@
     p {
       margin: 2px 0;
       text-align: left;
+      color: $light-mode-black;
+      opacity: 0.85;
+      font-family: Arial, sans-serif !important;
```

Comment:
This font is different from $font-family-base, is this on purpose?

Reply:
This is because of the blurrieness on the internet i found this should help it a little